web/main/colors.css

```diff
@@ -45,6 +45,9 @@
       0 -1px oklch(0% 0 0 / 20%);
     --field-color: oklch(100% 0 0);
     --field-shadow: inset 0 1px 2px oklch(from var(--shadow-color) l c h / 8%);
+    --link-color: var(--blue-700);
+    --link-color-visited: var(--violet-700);
```

Comment:
The color difference between visited and non-visited is too small

![Captura desde 2024-04-04 23-57-47](https://github.com/hplush/slowreader/assets/19343/8886c3c4-352e-42ba-9e80-c92f26422266)

Reply:
maybe use this one, probably looks good 
<img width="316" alt="Screenshot 2024-04-05 at 00 47 00" src="https://github.com/hplush/slowreader/assets/12436035/e24b9d5f-ea13-4199-b346-c98db7179d4c">

Reply:
Yes, perfect